cmd/bundle.rb

```diff
@@ -57,6 +57,8 @@ def bundle_args
              description: "`list` tap dependencies."
       switch "--mas",
              description: "`list` Mac App Store dependencies."
+      switch "--whalebrew",
```

Comment:
Also needs updates on https://github.com/Homebrew/homebrew-bundle/blob/05f7b1ac5c4a74b49ed46ef8d6ea2a51d8c198cb/cmd/bundle.rb#L12 and https://github.com/Homebrew/homebrew-bundle/blob/05f7b1ac5c4a74b49ed46ef8d6ea2a51d8c198cb/cmd/bundle.rb#L20

Reply:
good pick up! addressed in 9abddde